frontend-react/src/components/Admin/AdminReceiverDashboard.tsx

```diff
@@ -0,0 +1,775 @@
+                eachRow.filterOnCheckResultStr(filterErrorResults)
+        )
+    );
+
+    const showDetailsModal = (subData: AdmConnStatusDataType[]) => {
+        if (subData.length) {
+            setCurrentDataForModal(subData);
+            modalShowInfoRef?.current?.toggleModal(undefined, true);
+        }
+    };
+
+    return (
+        <section className="grid-container">
+            <h4>Receiver Status dashboard</h4>
+            <section>
+                CRON job results that check if receivers are working.
+            </section>
+            <form autoComplete="off" className="grid-row margin-0">
+                <div className="flex-auto">
+                    <DateRangePicker
+                        className={`${StyleClass.DATE_CONTAINER} margin-0`}
+                        startDateLabel="From (Start Range):"
+                        startDatePickerProps={{
+                            id: "start-date",
+                            name: "start-date-picker",
+                            defaultValue: defaultStartDateIso(),
+                            onChange: (s) =>
+                                setStartDate(
+                                    s
+                                        ? new Date(s).toISOString()
+                                        : defaultStartDateIso()
+                                ),
+                        }}
+                        endDateLabel="Until (End Range):"
+                        endDatePickerProps={{
+                            id: "end-date",
+                            name: "end-date-picker",
+                            defaultValue: defaultEndDateIso(),
+                            onChange: (s) =>
+                                setEndDate(
+                                    s ? new Date(s).toISOString() : undefined
+                                ),
+                        }}
+                    />
+                </div>
+                <div className="flex-fill margin-1">
+                    <Label
+                        className="font-sans-xs usa-label"
+                        htmlFor="input_filter_receivers"
+                    >
+                        Receiver:
+                    </Label>
+                    <TextInput
+                        id="input_filter_receivers"
+                        name="input_filter_receivers"
+                        type="text"
+                        autoComplete="off"
+                        aria-autocomplete="none"
+                        autoFocus
+                        onChange={(evt) => setFilterReceivers(evt.target.value)}
+                    />
+                </div>
+
+                <div className="flex-fill margin-1">
+                    <Label
+                        className="font-sans-xs usa-label"
+                        htmlFor="input_filter_errors"
+                    >
+                        Results:
+                    </Label>
+                    <TextInput
```

Comment:
it wasn't entirely clear to me what this filter was filtering on, also don't know how to make it more clear other than adding a tool tip? Is there a content / design review for this on the table?

Reply:
It's an admin-only page so there's no formal design or user testing since our internal O&O team are the only users. To date, the design team has been very busy with user-facing features, so their time is too valuable for the first pass n an internal-only dashboard. @drew-usds 

And you or an O&O team member are literally the user testing and I'm HAPPY to make any design changes

**Why it functions the way it does:** 
There is an error field that isn't showing in the table view and only shown in the details dialog because it's just too long. 

I played with having a single filter for both the visible data in the table (e.g. Org or Receiver) AND the buried error messages but it made the filter feel unpredictable. 

I can add hover help OR a tooltip or OR some better description text. Do you have a preference?

Reply:
Added `<Tooltip>` to give more UI discoverability